package.json

```diff
@@ -44,7 +44,7 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.1.2",
-    "axios": "^0.18.1",
```

Comment:
v19 is not compatible with axios-retry and causes infinite loop in case a retry is needed.